cmd/decode-mount-opts/decode-mount-opts.c

```diff
@@ -32,7 +32,7 @@ int main(int argc, char *argv[])
 		fprintf(stderr, "cannot parse given argument as a number\n");
 		return 1;
 	}
-	char buf[1000];
+	char buf[1000] = {0,};
```

Comment:
Why the extra comma and extra spaces (elsewhere)?  Can we use the simpler `{0}` as the convention?

Mechanical: `sed 's/= { *0,\? *};/= {0};/g'`

Reply:
I changed this now. Please note that it will still be `{ 0 }` (with the extra space) because our source code format tool (`make fmt`) will format it in this style. I removed the exta `,` as suggested.